src/ocm/hooks/useOpenshiftVersions.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { CpuArchitecture, OpenshiftVersionOptionType } from '../../common';
```

Comment:
What is the purpose of this file?

Reply:
I created some unit tests to validate that the new functions worked fine, and thought it would be great to have them tested.
But right now we don't have the test engine so in reality they wouldn't be executed.

So it's not useful unless we add the test engine back. Maybe we should create a ticket for it?

Reply:
Removed, we'll create a task to better handle this.